code/modules/projectiles/projectile/bullets.dm

```diff
@@ -124,6 +124,21 @@
 	damage = 10
 	weaken = 4
 	stun = 4
+	
+/obj/item/projectile/bullet/honker
+	damage = 0
+	weaken = 5
+	stun = 5
+	forcedodge = 1

```

Comment:
You're aware this means the bullet will pass through walls, machines and mobs, right? And with a range of 200 you're going to end up hearing that bikehorn sound a lot.